docs/pages/components/OptionsMenu.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: OptionsMenu
+description: A dropdown menu that shows a list of options when opened.
+source: https://github.com/dequelabs/cauldron/tree/develop/packages/react/src/components/OptionsMenu/OptionsMenu.tsx
+---
+
+import { OptionsMenu, OptionsMenuItem, OptionsMenuTrigger, Icon, Button } from '@deque/cauldron-react'
+
+```jsx
+import { 
+  OptionsMenu, 
+  OptionsMenuItem, 
+  OptionsMenuTrigger
+} from '@deque/cauldron-react'
+```
+
+## Examples
+
+### Default Trigger
+
+```jsx example
+<OptionsMenu
+  align="left"
+  trigger={triggerProps => (
+    <OptionsMenuTrigger {...triggerProps}>
+      <Icon type="kabob" label="Options" />
+    </OptionsMenuTrigger>
+  )}
+>
+  <OptionsMenuItem>This</OptionsMenuItem>
+  <OptionsMenuItem>That</OptionsMenuItem>
+  <OptionsMenuItem>The third</OptionsMenuItem>
+  <OptionsMenuItem disabled>And the other</OptionsMenuItem>
+</OptionsMenu>
+```
+
+### Alternate Trigger
+
+The trigger can be any type of component that acts as a button, allowing for advanced      customization of the trigger:
+
+```jsx example
+<OptionsMenu
+  align="left"
+  trigger={triggerProps => (
+    <Button variant="secondary" {...triggerProps}>
+      Options
+      <Icon type="arrow-circle-down" />
+    </Button>
+  )}
+>
+  <OptionsMenuItem>This</OptionsMenuItem>
+  <OptionsMenuItem>That</OptionsMenuItem>
+  <OptionsMenuItem>The third</OptionsMenuItem>
+  <OptionsMenuItem disabled>And the other</OptionsMenuItem>
+</OptionsMenu>
+```
+
+## Props
+<ComponentProps className={true} props={[
+  {
+    name: 'onClose',
+    type: 'function',
+    description: 'Function called when the OptionMenu component is closed.'
+  },
+  {
+    name: 'onSelect',
+    type: 'function',
+    description: 'Function called when a child of the OptionMenu component is selected.'
+  },
+  {
+    name: 'trigger',
+    type: 'function',
```

Comment:
Nothing to do with your PR, but it seems weird to me that this takes a function instead of a `ReactElement`

Reply:
It's a [render function/prop](https://react.dev/reference/react/cloneElement#passing-data-with-a-render-prop) because it needs to spread certain properties onto the component.

If we were to rewrite this today, we would likely use a hook instead to provide the same functionality.

Reply:
Ah, that makes sense